tests/helpers/integration_setup.py

```diff
@@ -40,11 +39,15 @@ def stop_service():
 
     if container_process is not None:
         logger.info('Stopping container')
-        container_process.send_signal(signal.SIGTERM)
+
+        # Stop and remove containers
+        cwd = 'tests/integration/docker'
+        subprocess.run("docker-compose --ansi never down", shell=True, check=True, cwd=cwd)
+
         logger.info('Waiting until service has stopped...')
 
-        if not common.wait_for_line("container.err",
-                                    lambda line: 'Stopping' in line and 'done' in line,
+        if not common.wait_for_line("container.out",
+                                    lambda line: "exited with code 0" in line,
```

Comment:
Same question - why the change?

Reply:
Running docker-compose down shuts the containers down gracefully, as confirmed by the presence of the "exited with code 0" message in the container.out file.

Reply:
Sending a sigterm also shuts the containers down gracefully. If a sigterm isn't enough, then docker is sending a sigkill which isn't a graceful shutdown

Reply:
When using container_process.send_signal(signal.SIGTERM), Docker will not automatically send a SIGKILL if the container does not stop. The behavior you described applies to `docker stop` and `docker-compose`.

Reply:
Right, what I'm saying is that the container shouldn't require a sigkill to stop - it should responsd to just a sigterm

Reply:
https://github.com/kbase/search_api2/pull/63#discussion_r2093594044